src/excalidraw-app/index.tsx

```diff
@@ -369,11 +362,6 @@ const ExcalidrawWrapper = () => {
       }
     };
 
-    const titleTimeout = setTimeout(
-      () => (document.title = APP_NAME),
-      TITLE_TIMEOUT,
-    );
-
```

Comment:
we still might want to keep it behind the time out
cc @dwelle

Reply:
don't recall why it was put under a timer to begin with? Does the `document.title` update call take long (by which I mean on the order of millis)?